### PR TITLE
Tweak: Update the controls and CSS variables in Container to logical properties [ED-11594]

### PR DESCRIPTION
**Pull request by Barcov**

### CODE REVIEW

File: assets/dev/scss/frontend/_container.scss [hunk 1]
```diff
@@ -14,29 +14,28 @@
...
-	--margin-left: 0;
-	--padding-top: var( --container-default-padding-top, 10px );
-	--padding-right: var( --container-default-padding-right, 10px );
-	--padding-bottom: var( --container-default-padding-bottom, 10px );
-	--padding-left: var( --container-default-padding-left, 10px );
+	--margin-block-start: 0;
+	--margin-inline-end: 0;
+	--margin-block-end: 0;
+	--margin-inline-start: 0;
+	--padding-block-start: var( --container-default-padding-block-start, 10px );
```
[the remark this answers is not in the document]
Comment (Barcov) [reply assets/dev/scss/frontend/_container.scss:26]: @hein-obox thanks for this suggestion. I've updated this

File: assets/dev/scss/frontend/_container.scss [hunk 2]
```diff
@@ -63,8 +62,8 @@
 	&-full,
 	& > .e-con-inner {
 		text-align: var( --text-align );
-		padding-top: var( --padding-top );
-		padding-bottom: var( --padding-bottom );
+		padding-block-start: var( --padding-block-start );
```
Comment (hein-obox) [inline assets/dev/scss/frontend/_container.scss:65]: Do we not need a possible fallback value for possible older Pro version?

File: modules/nested-tabs-html/widgets/nested-tabs-html.php [hunk 3]
```diff
@@ -399,7 +399,7 @@ protected function register_controls() {
 			}
 
 			$dropdown_options[ $breakpoint_key ] = sprintf(
-				/* translators: 1: Breakpoint label, 2: `>` character, 3: Breakpoint value. */
```
Comment (rami-elementor) [inline modules/nested-tabs-html/widgets/nested-tabs-html.php:402]: please revert, this is our convention.

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 4]
```diff
@@ -381,7 +381,7 @@ protected function register_controls() {
 			}
 
 			$dropdown_options[ $breakpoint_key ] = sprintf(
-				/* translators: 1: Breakpoint label, 2: `>` character, 3: Breakpoint value. */
+			    /* translators: 1: Breakpoint label, 2: `>` character, 3: Breakpoint value. */
```
Comment (rami-elementor) [inline modules/nested-tabs/widgets/nested-tabs.php:384]: revert

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 5]
```diff
@@ -33,26 +34,27 @@
...
-				border-width: var(--n-accordion-border-width);
-				border-color: var(--n-accordion-border-color);
-				border-style: var(--n-accordion-border-style);
-				justify-content: var(--n-accordion-title-justify-content);
-				border-radius: var(--n-accordion-border-radius);
-				flex-grow: var(--n-menu-title-flex-grow);
-				gap: var(--n-accordion-icon-gap);
-				color: var(--n-accordion-title-normal-color);
+				padding-block: var( --n-accordion-padding-block,  var( --n-accordion-padding ) );
+				padding-inline: var( --n-accordion-padding-inline, var( --n-accordion-padding ) );
```
Comment (rami-elementor) [inline modules/nested-accordion/assets/scss/frontend.scss:49]: the shorthand are supported by Safari 14.1, Elementor supports Safari 14.

ref: https://caniuse.com/mdn-css_properties_padding-block

File: modules/nested-tabs-html/widgets/nested-tabs-html.php (hunk 3 above)
Comment (Barcov) [reply modules/nested-tabs-html/widgets/nested-tabs-html.php:402]: @rami-elementor consider it done. Thanks for pointing this out.

File: modules/nested-accordion/assets/scss/frontend.scss (hunk 5 above)
Comment (Barcov) [reply modules/nested-accordion/assets/scss/frontend.scss]: @rami-elementor Thanks for pointing that out, I've removed all shorthand logical properties

File: includes/controls/groups/border.php [hunk 6]
```diff
@@ -73,12 +73,14 @@ protected function init_fields() {
 			],
 		];
 
+		// To do: remove after we update all widget border-width properties to logical.
+		$border_physical_properties = 'border-width: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};';
```
Comment (Barcov) [inline includes/controls/groups/border.php:77]: Fallback value for widgets in core and pro that don't yet implement logical Border.

Comment (hein-obox) [reply includes/controls/groups/border.php:77]: I think that we should remove this.
As far as I can see, this control doesn't use a CSS variable but will general the CSS properties directly. I don't think there is a need to create physical and logical properties at the same time.

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 7]
```diff
@@ -9,7 +9,10 @@
 	--n-accordion-item-title-flex-grow: initial;
 	--n-accordion-item-title-space-between: 0px;
 	--n-accordion-item-title-distance-from-content: 0px;
-	--n-accordion-padding: 10px;
+	--n-accordion-padding-block-start: 10px;
```
Comment (hein-obox) [inline modules/nested-accordion/assets/scss/frontend.scss:12]: Just for clarity, what is the problem with using the shorthand directly?
I am happy to use the shorthand, because it is quicker.
Not sure what Rami preferes.

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 8]
```diff
@@ -33,34 +36,45 @@
 			position: relative;
 
 			&:not(:last-child) {
-				margin-bottom: var(--n-accordion-item-title-space-between);
+				margin-block-end: var(--n-accordion-item-title-space-between);
 			}
 
 			:where( & > .e-con ) {
-				border: var(--n-accordion-border-width) var(--n-accordion-border-style) var(--n-accordion-border-color);
+				border-inline-start-width: var( --n-accordion-border-width );
```
Comment (hein-obox) [inline modules/nested-accordion/assets/scss/frontend.scss:43]: Same question as above.

Please remove the spaces here.

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 9]
```diff
@@ -33,34 +36,45 @@
...
-				padding: var(--n-accordion-padding);
-				border-width: var(--n-accordion-border-width);
-				border-color: var(--n-accordion-border-color);
-				border-style: var(--n-accordion-border-style);
-				justify-content: var(--n-accordion-title-justify-content);
-				border-radius: var(--n-accordion-border-radius);
-				flex-grow: var(--n-menu-title-flex-grow);
-				gap: var(--n-accordion-icon-gap);
-				color: var(--n-accordion-title-normal-color);
+				padding-block-start: var( --n-accordion-padding-block-start,  var( --n-accordion-padding ) );
```
Comment (hein-obox) [inline modules/nested-accordion/assets/scss/frontend.scss:54]: Same question as above.

Remove spaces.

File: modules/nested-tabs-html/widgets/nested-tabs-html.php (hunk 3 above)
Comment (hein-obox) [reply modules/nested-tabs-html/widgets/nested-tabs-html.php:402]: Not reverted?

File: modules/nested-tabs-html/widgets/nested-tabs-html.php [hunk 10]
```diff
@@ -1108,15 +1105,15 @@ protected function render_tab_titles_html( $item_settings ): string {
 
 		ob_start();
 		?>
-			<button <?php echo $render_attributes; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
+        <button <?php echo $render_attributes; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>>
```
Comment (hein-obox) [inline modules/nested-tabs-html/widgets/nested-tabs-html.php:1108]: Restore original indentation.

File: assets/dev/scss/frontend/_container.scss [hunk 11]
```diff
@@ -134,40 +133,44 @@
 		gap: var( --gap );
 		width: 100%;
 		max-width: var( --content-width );
-		margin: 0 auto;
-		padding-inline: 0;
+		margin-inline-start: auto;
```
Comment (hein-obox) [inline assets/dev/scss/frontend/_container.scss:136]: Please revert changes back to 'margin: 0 auto;'

File: includes/controls/groups/border.php [hunk 12]
```diff
@@ -73,12 +73,14 @@ protected function init_fields() {
...
 
+		// To do: remove after we update all widgets border-width properties to logical.
+		$border_physical_properties = 'border-width: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};';
 		$fields['width'] = [
 			'label' => esc_html_x( 'Width', 'Border Control', 'elementor' ),
 			'type' => Controls_Manager::DIMENSIONS,
 			'size_units' => [ 'px', 'em', 'rem', 'vw', 'custom' ],
 			'selectors' => [
-				'{{SELECTOR}}' => 'border-width: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+				'{{SELECTOR}}' => "$border_physical_properties border-block-start-width: {{TOP}}{{UNIT}}; border-inline-start-width {{RIGHT}}{{UNIT}}; border-block-end-width {{BOTTOM}}{{UNIT}}; --border-inline-start-width {{LEFT}}{{UNIT}};",
```
Comment (hein-obox) [inline includes/controls/groups/border.php:83]: I believe we can revert the changes, I don't see a problem with the original code.

File: modules/nested-accordion/assets/scss/frontend.scss [hunk 13]
```diff
@@ -33,34 +36,45 @@
...
+				padding-block-end: var( --n-accordion-padding-block-end,  var( --n-accordion-padding ) );
+				padding-inline-start: var( --n-accordion-padding-inline-start, var( --n-accordion-padding ) );
+				padding-inline-end: var( --n-accordion-padding-inline-end, var( --n-accordion-padding ) );
+				border-inline-start-width: var( --n-accordion-border-width );
```
Comment (hein-obox) [inline modules/nested-accordion/assets/scss/frontend.scss:58]: Revert changes?

File: modules/nested-accordion/widgets/nested-accordion.php [hunk 14]
```diff
@@ -418,7 +418,7 @@ private function add_accordion_style_section() {
 				'type' => Controls_Manager::DIMENSIONS,
 				'size_units' => [ 'px', '%', 'em', 'rem', 'custom' ],
 				'selectors' => [
-					'{{WRAPPER}} ' => '--n-accordion-padding: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+					'{{WRAPPER}}' => '--n-accordion-padding-block-start: {{TOP}}{{UNIT}}; --n-accordion-padding-block-end: {{BOTTOM}}{{UNIT}}; --n-accordion-padding-inline-start: {{RIGHT}}{{UNIT}}; --n-accordion-padding-inline-end: {{LEFT}}{{UNIT}};',
```
Comment (hein-obox) [inline modules/nested-accordion/widgets/nested-accordion.php:421]: Revert changes?

File: modules/nested-tabs-html/assets/scss/frontend.scss [hunk 15]
```diff
@@ -164,7 +164,10 @@
 			flex-direction: var( --n-tabs-title-direction );
 			justify-content: var( --n-tabs-title-justify-content-toggle, var( --n-tabs-title-justify-content ) );
 			gap: var( --n-tabs-icon-gap );
-			border-width: var( --n-tabs-border-width );
+			border-inline-start-width: var( --n-tabs-border-width );
```
Comment (hein-obox) [inline modules/nested-tabs-html/assets/scss/frontend.scss:167]: Revert changes?

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 16]
```diff
@@ -108,30 +104,33 @@
 			flex-direction: var( --n-tabs-title-direction );
 			justify-content: var( --n-tabs-title-justify-content-toggle, var( --n-tabs-title-justify-content ) );
 			gap: var( --n-tabs-icon-gap );
-			border-width: var( --n-tabs-border-width );
+			border-inline-start-width: var( --n-tabs-border-width );
```
Comment (hein-obox) [inline modules/nested-tabs/assets/scss/frontend.scss:107]: Revert changes?

File: modules/nested-tabs/widgets/nested-tabs.php [hunk 17]
```diff
@@ -381,7 +381,7 @@ protected function register_controls() {
 			}
 
 			$dropdown_options[ $breakpoint_key ] = sprintf(
-				/* translators: 1: Breakpoint label, 2: `>` character, 3: Breakpoint value. */
+			/* translators: 1: Breakpoint label, 2: `>` character, 3: Breakpoint value. */
```
Comment (hein-obox) [inline modules/nested-tabs/widgets/nested-tabs.php:384]: Revert change

File: tests/playwright/sanity/modules/nested-tabs-html/nested-tabs-html.test.ts [hunk 18]
```diff
@@ -1163,7 +1169,10 @@ test.describe( 'Nested Tabs tests @nested-tabs-html', () => {
 						await expect.soft( activeContainer ).toHaveCSS( 'box-shadow', expectedCssValue );
 						break;
 					case 'containerPadding':
-						await expect.soft( activeContainer ).toHaveCSS( 'padding', expectedCssValue );
+						await expect.soft( activeContainer ).toHaveCSS( 'padding-inline-start', expectedCssValue );
```
Comment (hein-obox) [inline tests/playwright/sanity/modules/nested-tabs-html/nested-tabs-html.test.ts:1172]: Revert test

File: includes/controls/groups/border.php (hunk 6 above)
Comment (Barcov) [reply includes/controls/groups/border.php:77]: Mega Menu overrides border-width in the control. Other Nested Elements use logical defaults set in the border control.

There are several widgets beyond the scope of this PR that we need to check when we flip to logical CSS on the border control. This is the reason I'm proposing fallback values until we have updated tested. 

### Widgets possibly affected:

#### Core:
- testimonial
- column
- container
- section
- common
- image-carousel
- image-gallery
- image
- social-icons
- testimonial
- button (button trait)

#### Pro:
- blockquote
- call-to-action
- countdown
- flip-box
- form
- login
- taxonomy-filter
- nav-menu
- popup
- price-table
- Theme Builder
- Woocommerce & Woocommerce widgets
- popup

File: includes/controls/groups/border.php (hunk 12 above)
Comment (Barcov) [reply includes/controls/groups/border.php:83]: @hein-obox  reverting, heads up this is tied to logical border props in Mega Menu around line 1835 & the same in Mega Menu html

File: modules/nested-accordion/assets/scss/frontend.scss (hunk 7 above)
Comment (Barcov) [reply modules/nested-accordion/assets/scss/frontend.scss:12]: Padding here is set on all 4 sides by the NA padding control and would become incompatible with the intended LTR / RTL tweak we're introducing if I revert this change.

File: includes/controls/groups/border.php (hunk 12 above)
Comment (hein-obox) [reply includes/controls/groups/border.php:83]: I would like to revert this change.

File: modules/nested-accordion/widgets/nested-accordion.php [hunk 19]
```diff
@@ -418,7 +418,7 @@ private function add_accordion_style_section() {
 				'type' => Controls_Manager::DIMENSIONS,
```
Comment (hein-obox) [inline modules/nested-accordion/widgets/nested-accordion.php:418]: The 'content_padding' control should be updated.
I believe the original screenshots were correct, and should be reverted.

File: modules/nested-accordion/assets/scss/frontend.scss (hunk 7 above)
Comment (Barcov) [reply modules/nested-accordion/assets/scss/frontend.scss:12]: Reverting, Moved this to a separate PR.

File: modules/nested-accordion/widgets/nested-accordion.php (hunk 19 above)
Comment (Barcov) [reply modules/nested-accordion/widgets/nested-accordion.php]: Reverted

File: modules/nested-tabs-html/widgets/nested-tabs-html.php (hunk 3 above)
Comment (Barcov) [reply modules/nested-tabs-html/widgets/nested-tabs-html.php:402]: Yes it's been reverted. PR seems stuck, I'll check again after pushing other changes to this file.

File: modules/nested-tabs-html/widgets/nested-tabs-html.php [hunk 20]
```diff
@@ -1206,16 +1203,17 @@ protected function render() {
 		<div <?php $this->print_render_attribute_string( 'elementor-tabs' ); ?>>
 			<div class="e-n-tabs-heading" role="tablist">
 				<?php echo $tab_titles_html;// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
-			</div>
-			<div class="e-n-tabs-content">
+            </div>
```
Comment (Barcov) [inline modules/nested-tabs-html/widgets/nested-tabs-html.php]: Tabs converted to spaces - hunting to see if this is an Storm bug or caused on CI. Converting spaces back to tabs.

File: modules/nested-tabs/widgets/nested-tabs.php (hunk 17 above)
Comment (Barcov) [reply modules/nested-tabs/widgets/nested-tabs.php:384]: Reverted
image

File: tests/playwright/sanity/modules/nested-accordion/nested-accordion-style.test.ts-snapshots/nested-Accordion-content-style-front-linux.png
Comment (Barcov) [inline tests/playwright/sanity/modules/nested-accordion/nested-accordion-style.test.ts-snapshots/nested-Accordion-content-style-front-linux.png:1]: Reverted

File: tests/playwright/sanity/modules/nested-accordion/nested-accordion-style.test.ts-snapshots/nested-Accordion-content-style-override-linux.png
Comment (Barcov) [inline tests/playwright/sanity/modules/nested-accordion/nested-accordion-style.test.ts-snapshots/nested-Accordion-content-style-override-linux.png:1]: Reverted

File: modules/nested-tabs-html/assets/scss/frontend.scss [hunk 21]
```diff
@@ -82,35 +82,35 @@
...
 
 	.e-n-tabs {
-		display: var( --n-tabs-display );
-		flex-direction: var( --n-tabs-direction );
-		gap: var( --n-tabs-gap );
-		text-align: $start;
+		display: var(--n-tabs-display);
+		flex-direction: var(--n-tabs-direction);
+		gap: var(--n-tabs-gap);
+		text-align: start;
```
Comment (Barcov) [inline modules/nested-tabs-html/assets/scss/frontend.scss:102]: Directional `start`

File: modules/nested-accordion/widgets/nested-accordion.php [hunk 22]
```diff
@@ -418,7 +418,7 @@ private function add_accordion_style_section() {
...
 				'size_units' => [ 'px', '%', 'em', 'rem', 'custom' ],
 				'selectors' => [
-					'{{WRAPPER}} ' => '--n-accordion-padding: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+					'{{WRAPPER}} ' => '--padding-block-start: {{TOP}}{{UNIT}}; --padding-inline-end: {{RIGHT}}{{UNIT}}; --padding-block-end: {{BOTTOM}}{{UNIT}}; --padding-inline-start: {{LEFT}}{{UNIT}};',
```
Comment (hein-obox) [inline modules/nested-accordion/widgets/nested-accordion.php:421]: Please reverse.

File: includes/elements/container.php [hunk 23]
```diff
@@ -1009,7 +1009,7 @@ protected function register_border_controls() {
 				'fields_options' => [
```
Comment (hein-obox) [inline includes/elements/container.php:1009]: I am still seeing physical properties.
e.g. 
--border-top-left-radius

File: modules/nested-accordion/widgets/nested-accordion.php [hunk 24]
```diff
@@ -418,7 +418,7 @@ private function add_accordion_style_section() {
 				'type' => Controls_Manager::DIMENSIONS,
 				'size_units' => [ 'px', '%', 'em', 'rem', 'custom' ],
```
Comment (hein-obox) [inline modules/nested-accordion/widgets/nested-accordion.php:419]: Update: --border-bottom-right-radius etc

File: includes/elements/container.php (hunk 23 above)
Comment (Barcov) [reply includes/elements/container.php:1009]: These CSS variables are not implemented in the container SCSS. 
I've also checked `main`.

Have looks like these variables have been orphaned since they where created and replaced with 
`--border-radius`

Was `--border-top-left-radius` (right|top|bottom)  a fallback meant to be removed later?
Or have we dropped functionality on another Merged PR by mistake?

File: modules/nested-accordion/widgets/nested-accordion.php (hunk 24 above)
Comment (Barcov) [reply modules/nested-accordion/widgets/nested-accordion.php:419]: These variables have been removed from _container.scss. This override no longer does anything as far as I can tell. Unless there's some BC angle in play that I'm not seeing.

File: includes/elements/container.php (hunk 23 above)
Comment (hein-obox) [reply includes/elements/container.php:1026]: Can you please delete these variables inside container.php and inside nested-accordion.php?

File: modules/nested-accordion/widgets/nested-accordion.php [hunk 25]
```diff
@@ -418,7 +418,7 @@ private function add_accordion_style_section() {
...
 				'selectors' => [
-					'{{WRAPPER}} ' => '--n-accordion-padding: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
+					'{{WRAPPER}} ' => 'n-accordion-padding: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
```
Comment (hein-obox) [inline modules/nested-accordion/widgets/nested-accordion.php:421]:
```suggestion
					'{{WRAPPER}} ' => '--n-accordion-padding: {{TOP}}{{UNIT}} {{RIGHT}}{{UNIT}} {{BOTTOM}}{{UNIT}} {{LEFT}}{{UNIT}};',
```

File: modules/nested-accordion/widgets/nested-accordion.php [hunk 26]
```diff
@@ -803,50 +803,50 @@ protected function content_template() {
...
 			<# if ( settings['items'] ) {
 			const elementUid = view.getIDInt().toString().substring( 0, 3 ),
-				titleHTMLTag = elementor.helpers.validateHTMLTag( settings.title_tag ),
-				itemTitleText = 'item-title-text-' + elementUid,
-				defaultState = settings.default_state,
-				itemTitleIcon = elementor.helpers.renderIcon( view, settings['accordion_item_title_icon'], { 'aria-hidden': true }, 'i', 'object' ) ?? '',
-				itemTitleIconActive = '' === settings.accordion_item_title_icon_active.value
-					? itemTitleIcon
-					: elementor.helpers.renderIcon( view, settings['accordion_item_title_icon_active'], { 'aria-hidden': true }, 'i', 'object' );
+			titleHTMLTag = elementor.helpers.validateHTMLTag( settings.title_tag ),
```
Comment (hein-obox) [inline modules/nested-accordion/widgets/nested-accordion.php:806]: Revert indentation.

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 27]
```diff
@@ -8,17 +8,17 @@
 
 	&,
 	& a {
-		color: var( --n-tabs-title-color-active );
```
Comment (Barcov) [inline modules/nested-tabs/assets/scss/frontend.scss:11]: Can we remove spaces in variables.

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 28]
```diff
@@ -8,17 +8,17 @@
...
+		color: var(--n-tabs-title-color-active);
```
Comment (hein-obox) [inline modules/nested-tabs/assets/scss/frontend.scss:11]: I believe we did that already?

Screenshot 2023-08-18 at 16 39 27

File: modules/nested-tabs/assets/scss/frontend.scss (hunk 27 above)
Comment (hein-obox) [reply modules/nested-tabs/assets/scss/frontend.scss:11]: See below

**Review by hein-obox — APPROVED** [review]

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 29]
```diff
@@ -330,19 +330,19 @@
...
-				margin-block-start: var( --n-tabs-title-gap );
+				margin-block-start: var(--n-tabs-title-gap);
 			}
 
 			&[aria-selected="true"] {
-				margin-block-end: var( --n-tabs-gap );
+				margin-block-end: var(--n-tabs-gap);
 			}
 		}
 	}
```
Comment (Barcov) [inline modules/nested-tabs/assets/scss/frontend.scss:348]: Notice Breakpoint none was removed.  Was this intentional?

```
// Breakpoint - none.
.e-n-tabs-none {
	> .elementor-widget-container > .e-n-tabs > .e-n-tabs-content > .e-collapse {
		display: none;
	}
}
```

**Review by davseve — APPROVED** [review]

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 30]
```diff
@@ -160,64 +160,64 @@
...
+				transform var(--n-tabs-title-transition);
 
 			@include button-focus-outline;
 
 			& span svg,
 			& span i {
-				transition: color var( --n-tabs-title-transition ),
-					fill var( --n-tabs-title-transition );
+				transition: color var(--n-tabs-title-transition ),
+					fill var(--n-tabs-title-transition);
```
Comment (TzviRabinovitch) [inline modules/nested-tabs/assets/scss/frontend.scss:198]:
```suggestion
				fill var(--n-tabs-title-transition);
```

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 31]
```diff
@@ -160,64 +160,64 @@
...
+			padding-block-start: var(--n-tabs-title-padding-block-start);
+			padding-inline-end: var(--n-tabs-title-padding-inline-end);
+			padding-block-end: var(--n-tabs-title-padding-block-end);
+			padding-inline-start: var(--n-tabs-title-padding-inline-start);
+			border-radius: var(--n-tabs-title-border-radius);
+			height: var(--n-tabs-title-height);
+			width: var(--n-tabs-title-width);
+			white-space: var(--n-tabs-title-white-space);
+			transition: background var(--n-tabs-title-transition ),
+				color var(--n-tabs-title-transition ),
```
Comment (TzviRabinovitch) [inline modules/nested-tabs/assets/scss/frontend.scss:183]:
```suggestion
			color var(--n-tabs-title-transition ),
```

**Review by TzviRabinovitch — APPROVED** [review]

**Review by davseve — APPROVED** [review]
